src/ImageService.Data.Provider.MsSql.Ef/Migrations/20210820233300_InitialTable.cs

```diff
@@ -12,7 +12,7 @@ class InitialTable : Migration
         protected override void Up(MigrationBuilder builder)
         {
             builder.CreateTable(
-                name: DbImagesUser.TableName,
+                name: DbImageUser.TableName,
```

Comment:
Do you understand what all developers who already ruined the service must remove Db?

Reply:
the name is the same

Reply:
but yes, it can broke other branches, wil fix it in other task